lib/rules/no-unknown-property.js

```diff
@@ -193,12 +193,9 @@ function getStandardName(name) {
   if (SVGDOM_ATTRIBUTE_NAMES[name]) {
     return SVGDOM_ATTRIBUTE_NAMES[name];
   }
-  let i;
-  const found = DOM_PROPERTY_NAMES.some((element, index) => {
```

Comment:
i'd rather keep this the same and allow undefined

Reply:
Updated